routes/users.js

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable prefer-destructuring */
+/* eslint-disable no-console */
+const express = require('express');
+const crypto = require('crypto');
+const bodyParser = require('body-parser');
+
+const cryptoAlgorithm = 'aes-256-ctr';
+const cryptoPassword = '4g632jzrb26DETTsr20ls7v12bMWmERC';
```

Comment:
crypto information should be stored in .env

Reply:
I'll look into it

Reply:
Again fixed this issue in latest commit